projects/core/src/auth/user-auth/facade/auth.service.ts

```diff
@@ -87,10 +90,18 @@ export class AuthService {
    */
   async loginWithCredentials(userId: string, password: string): Promise<void> {
     try {
+      let uId: string = userId;
```

Comment:
Do we _really_ need to wrap our new logic inside the `try catch` block? If not, let's move it up, outside of the block.